### PR TITLE
refactoring and adding approximant option for duration

### PR DESCRIPTION
**Pull request by tdent**
Should not change any existing functionality.  Enabling SEOBNRv2 duration function for backward compatibility

### CODE REVIEW

**Comment by tdent** [comment]
Any comments on this?

**Comment by titodalcanton** [comment]
Looks good to me Tom. It also looks like you can remove some pnutils imports now.

File: pycbc/events/trigger_fits.py [hunk 1]
```diff
@@ -211,3 +212,51 @@ def cdf_fn(x):
...
+    elif par in pnutils.named_frequency_cutoffs.keys():
+        parvals = pnutils.frequency_cutoff_from_name(par, m1, m2, s1z, s2z)
+    else:
+        # try asking for a LALSimulation frequency function
+        parvals = pnutils.get_freq(par, m1, m2, s1z, s2z)
+    return parvals
+
+
+def which_bin(par, minpar, maxpar, nbins, log=False):
+    """
```
Comment (ahnitz) [inline pycbc/events/trigger_fits.py]: Can you use a bit more standard formatting for the docstring?

File: pycbc/events/trigger_fits.py [hunk 2]
```diff
@@ -211,3 +212,51 @@ def cdf_fn(x):
...
+def get_masses(bank, tid):
+    """Helper function"""
+    m1 = bank['mass1'][:][tid]
+    m2 = bank['mass2'][:][tid]
+    s1z = bank['spin1z'][:][tid]
+    s2z = bank['spin2z'][:][tid]
+    return m1, m2, s1z, s2z
+
+
+def get_param(par, args, m1, m2, s1z, s2z):
```
Comment (ahnitz) [inline pycbc/events/trigger_fits.py]: Can this docstring have a bit more description?

File: pycbc/events/trigger_fits.py [hunk 3]
```diff
@@ -211,3 +212,51 @@ def cdf_fn(x):
         return 1 - cum_fndict[distr](x, alpha, thresh)
     return kstest(vals, cdf_fn)
 
+
+def get_masses(bank, tid):
```
Comment (ahnitz) [inline pycbc/events/trigger_fits.py]: I won't hold up merger for this, but this seems an odd function. Do you really save much by avoiding the duplication in this case?

**Pull request merged**